testing_and_setup/compass/ocean/global_ocean/scripts/create_E3SM_coupling_files.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python
+"""
+This script creates coupling files needed to run an MPAS-Ocean and
+MPAS-seaice within E3SM.
+
+Load the lastest e3sm-unified conda package.
+
+"""
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
```

Comment:
If we decide we're no longer supporting python 2.7, this could be dropped.

Reply:
Agreed. I'm removing it now.